src/home/homectl.c

```diff
@@ -2673,20 +2673,16 @@ static int parse_argv(int argc, char *argv[]) {
                                 break;
                         }
 
-                        if (!env_assignment_is_valid(optarg))
-                                return log_error_errno(SYNTHETIC_ERRNO(EINVAL),
-                                                       "Environment assignment '%s' not valid.", optarg);
```

Comment:
previously we'd verify the assignment in full, including character set and stuff, you dropped that without replacement?